README.md

```diff
@@ -199,6 +199,39 @@ const response = await replicate.models.get(model_owner, model_name);
 }
 ```
 
+### `replicate.models.list`
+
```

Comment:
It seems like we should have a one-liner description here below the heading, but there's no precedent for that in the existing docs. What do you think @mattt ?

Reply:
I'm all for it!